### PR TITLE
add live radio fixture data

### LINKED ISSUE #2754: Add fixture data for liveradio pages
**Is your feature request related to a problem? Please describe.**
We have requested that for consistency Ares should return static data for the live radio pages. This data should mimic the necessary values we need in the data response to build the full page. This will be some but not all of the fields specified in:
- [metadata](https://github.com/bbc/simorgh/blob/latest/data/afrique/frontpage/index.json#L2-L41)
- [promo](https://github.com/bbc/simorgh/blob/latest/data/afrique/frontpage/index.json#L4215-L4221)
- [relatedContent](https://github.com/bbc/simorgh/blob/latest/data/afrique/frontpage/index.json#L4222-L4236)
- content - see a data payload

**Describe the solution you'd like**
Add a fixture data asset for each service_id under the matching service, for liveradio.

**Describe alternatives you've considered**
N/a

**Testing notes**
[Tester to complete]

Dev insight: No testing required 

**Additional context**

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves https://github.com/bbc/simorgh/issues/2754

**Overall change:** Adds fixture data for live radio services Afaan Oromoo, Amharic, Indonesian, Korean, Tigrinya.

**Code changes:**

- Added afaanoromoo fixture data to `data/afaanoromoo/bbc_afaanoromo_radio/liveradio.json`
- Added amharic fixture data to `data/amharic/bbc_amharic_radio/liveradio.json`
- Added indonesia fixture data to `data/indonesia/bbc_indonesian_radio/liveradio.json`
- Added korean fixture data to `data/korean/bbc_korean_radio/liveradio.json`
- Added tigrinya fixture data to `data/tigrinya/bbc_tigrinya_radio/liveradio.json`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

**Testing notes**
No testing required

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]
This might be the PR to fix that annoying typo in the Afaan Oromoo directory name

File: data/afaanoromoo/bbc_afaanoromo_radio/liveradio.json [hunk 1]
```diff
@@ -1,29 +1,64 @@
...
+        "text": "Dhaggeeffadhaa",
+        "markupType": "plain_text",
+        "type": "heading"
+      },
+      { "text": "Sagantaawwan keenya.", "type": "paragraph" },
+      {
+        "id": "liveradio",
+        "subType": "primary",
+        "format": "audio",
+        "externalId": "bbc_afaanoromoo_radio",
```
Comment (pjlee11) [inline data/afaanoromoo/bbc_afaanoromo_radio/liveradio.json]:
```suggestion
        "externalId": "bbc_oromo_radio",
```

If we are having the mapping for `bbc_afaanoromoo_radio` to `bbc_oromo_radio` in simorgh this is likely where we would want to change it

File: data/korean/bbc_korean_radio/liveradio.json [hunk 2]
```diff
@@ -1,29 +1,320 @@
-{
+[
+  {
     "metadata": {
-        "id": "urn:bbc:ares::asset:korean/bbc_korean_radio/liveradio",
-        "tags": {}
+      "id": "urn:bbc:ares::ws_live:bbc_indonesian_radio",
```
Comment (ryanmccombe) [inline data/korean/bbc_korean_radio/liveradio.json]: This file seems to be an array of all configs rather than Korean

**Review by ryanmccombe — CHANGES_REQUESTED** [review]
https://github.com/bbc/simorgh/pull/3090#discussion_r312992300

**Comment by jroebu14** [comment]
> This might be the PR to fix that annoying typo in the Afaan Oromoo directory name

I think this is the PR that fixed that https://github.com/bbc/simorgh/pull/2947#pullrequestreview-271826277

**Review by ryanmccombe — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Pull request merged**